src/runners/handlers/slack.py

```diff
@@ -67,19 +66,31 @@ def handle(
     slack_token = vault.decrypt_if_encrypted(slack_token_ct)
 
     sc = SlackClient(slack_token)
-
     # otherwise we will retrieve email from assignee and use it to identify Slack user
     # Slack user id will be assigned as a channel
 
     title = alert['TITLE']
 
     if recipient_email is not None:
-        result = sc.api_call("users.lookupByEmail", email=recipient_email)
+        if isinstance(recipient_email, str):
+                user_object = sc.api_call("users.lookupByEmail", email=recipient_email)
+                user = user_object['user']
+                users.append(user['id'])
+                result = sc.api_call("conversations.open", users=users)
+
+
+        else:
+            users = []
+            for email in recipient_email:
+                user_object = sc.api_call("users.lookupByEmail", email=email)
+                user = user_object['user']
```

Comment:
both here and above, when a user is not found, we will likely throw an exception without details about which user it err'd on. it might help error a little more gracefully via —

~~~py
assert 'user' in user_object, f'error while finding user "{email}"'
~~~